modules/test-ui/src/App.tsx

```diff
@@ -77,21 +70,27 @@ function App() {
         setChannel(channelRes.getValue());
       }
       setNode(client);
-      localStorage.setItem("mnemonic", mnemonic);
-      setMnemonic(mnemonic);
       client.on(EngineEvents.DEPOSIT_RECONCILED, async data => {
         console.log("Received EngineEvents.DEPOSIT_RECONCILED: ", data);
         await updateChannel(client, data.channelAddress);
       });
+      // TODO: this is required bc the event handlers are keyed on Date.now()
+      await delay(10);
```

Comment:
Is this still required with the modified `+ extra`?

Reply:
What's the `+ extra`?